sourcecode/apis/contentauthor/.env.defaults

```diff
@@ -9,10 +9,14 @@ DB_PASSWORD=mysqlpassword
 
```

Comment:
I dont think it is a good idea to put this file as default. It was specifically created for the local environment and could break things in the live environments

Reply:
This is also true of `.env.example`, which was previously copied to `.env` upon install and used for default values. `.env.defaults` is intended to replace it. Perhaps the best thing to do here is make sure nothing is declared here that wasn't previously declared in `.env.examples`?